frontend/www/js/omegaup/problem/creator/modules/cases.ts

```diff
@@ -225,6 +228,83 @@ export const casesStore: Module<CasesState, RootState> = {
         (line) => line.lineID !== lineIDToBeDeleted,
       );
     },
+    editLineValue(state, [lineID, value]: [LineID, string]) {
+      const selectedGroup = state.groups.find(
+        (group) => group.groupID === state.selected.groupID,
+      );
+      if (selectedGroup === undefined) return;
```

Comment:
```suggestion
      if (!selectedGroup) return;
```
Does this work? If yes, use it in the other if statements.